ansible/tasks/packages/main.yml

```diff
@@ -35,7 +35,6 @@
       - aria2
       - aspell
       - aspell-en
-      - at
```

Comment:
make a task to remove `at`, as well as `cron` and `systemd-cron` (and remove systemd-cron from packages list)

Reply:
nevermind, just saw the one to remove `at`, but the statement still holds for the crons